src/components/Plugins/index.jsx

```diff
@@ -0,0 +1,39 @@
+import { Chart } from 'react-chartjs-2';
+
+const registerTooltipPlugin = () => {
+  Chart.pluginService.register({
+    id: 'tooltipPlugin',
+    beforeRender(chart) {
+      this.pluginTooltips = [];
+      const globalChartObj = Chart;
+      // Process only if a data point was clicked on current chart object
+      if (globalChartObj.lastClickedDataPoint) {
+        // eslint-disable-next-line no-underscore-dangle
+        if (globalChartObj.lastClickedDataPoint._chart.chart.id !== chart.id) {
+          return;
+        }
+        // eslint-disable-next-line no-underscore-dangle
+        const datasetIndex = globalChartObj.lastClickedDataPoint._datasetIndex;
```

Comment:
I think it's fine that you created that two variables `index` and `datasetIndex` but you can also do this in one line: `const { _datasetIndex, _index } = globalChartObj.lastClickedDataPoint;` and if you wanted to rename them, like this ` const { datasetIndex: _datasetIndex, index: _index } ...`

Reply:
@sarah-clements - I used this format and it is not working. Also, unable to find any documentation regarding single line multiple variable declaration.

Reply:
Sorry, I got the key/value order mixed up. Should be like this: `const { _datasetIndex: datasetIndex, _index: index } = globalChartObj.lastClickedDataPoint;`. It's called [destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment).

Reply:
Thanks :)